static/js/github-configuration.js

```diff
@@ -1,30 +1,30 @@
 /* globals $ */
-$('.install-link').click(function (event) {
```

Comment:
There are a lot of small refactorings in this file and some others that don't seem to have anything to do with the goal of this PR. Could you revert those in this PR (and then maybe raise a second PR with just the refactorings? That will make it easier to review and merge.